.github/workflows/crucible-tracking.yaml

```diff
@@ -0,0 +1,118 @@
+name: crucible-tracking
+
+on:
+  workflow_call:
+    inputs:
+      app_id:
+        required: true
+        type: string
+    secrets:
+      private_key:
+        required: true
+
+jobs:
+  project-tracking:
+    runs-on: [ self-hosted, project-tracker ]
```

Comment:
why self-hosted ?

Reply:
I'm getting a bit tricky here.  I created a small self-hosted runner that will only be used to process these project management jobs, which are very lightweight, so that they never get blocked behind the bigger, long running, integration/unit test jobs.

Reply:
I should also note that I added the `project-tracker` tag to this runner to signify that it has the GitHub CLI tool installed.  I may add that tool to the other self-hosted runners in the future if necessary* which would allow us to (potentially**) run multiple jobs in parallel.

* This would really only be necessary if we start to observe a slow down in the processing of these jobs because there are many firing in parallel.  This would only happen if we start to have lots of issues and PRs submitted at the same time.

** I say potentially because the other runners could be blocked doing the existing integration/unit test jobs.